### PR TITLE
add docker-entrypoint.sh for using wildcards etc.

### PR DESCRIPTION
**Pull request by shakeme**
This will solve issue: https://github.com/mthenw/frontail/issues/88

### CODE REVIEW

**Review by mthenw — COMMENTED** [review]
Hey, thanks for the PR!

File: Dockerfile [hunk 1]
```diff
@@ -1,9 +1,10 @@
 FROM mhart/alpine-node:6.14.2
 
-RUN apk update && apk upgrade && apk add --no-cache git
+RUN apk add --no-cache git
```
Comment (mthenw) [inline Dockerfile]: What's the reason for this change?

Comment (shakeme) [reply Dockerfile]: Hi. It seams that your base image is up to date so i thought it's ok to just install the dependencies without keeping the package cache from "apk update". If you still want to update the whole image, "apk --no-cache upgrade && apk add --no-cache git" would be a nice alternative.

Comment (mthenw) [reply Dockerfile]: Sounds good, can you make the change?

Comment (shakeme) [reply Dockerfile]: Done.

**Comment by mthenw** [comment]
Thanks, looks great!
